provision/openshift/isup.sh

```diff
@@ -5,7 +5,9 @@
 KC_NAMESPACE_PREFIX=$(cat .task/var-KC_NAMESPACE_PREFIX)
 KC_HOSTNAME_SUFFIX=$(cat .task/var-KC_HOSTNAME_SUFFIX)
 
-source .env
+if [ -f .env ]; then
+  source .env
+fi
```

Comment:
```suggestion
if [ -f ./.env ]; then
  source ./.env
fi
```

Reply:
@kami619 doesn't really matter as the two are identical, but can change it as above to make it consistent with the other scripts

Reply:
yup, called it out for consistency sake 💯